### ISSUE TITLE
[BUG] predicting with yolov8 model throws error "object of type 'int' has no len()"

### ISSUE THREAD

**Issue by stevensagaert** [voxel51/fiftyone#4741]
### Describe the problem

running predictions an a dataset with an ultralytics yolo model gives error "object of type 'int' has no len()"

### Code to reproduce issue

#load a dataset (custom or from zoo)
dataset=...

import fiftyone.zoo as foz
#either of these models gives the same error
model = foz.load_zoo_model("yolov8m-seg-coco-torch")
#model = foz.load_zoo_model("yolov8m-coco-torch")

dataset.apply_model(model, label_field="predictions")

```
# commands and/or screenshots here
```

### System information

-   **OS Platform and Distribution** (e.g., Linux Ubuntu 22.04): Linux Ubuntu 22.04
-   **Python version** (`python --version`): 3.10.12
-   **FiftyOne version** (`fiftyone --version`): v0.25.0
-   **FiftyOne installed from** (pip or source): pip

### Other info/logs

this is the fix:
in utils/ultralytics.py:
def _extract_track_ids(result):
    """Get ultralytics track ids if present, else use Nones"""
    return (
        result.boxes.id.detach().cpu().numpy().astype(int)
        if result.boxes.is_track
        else [None] * result.boxes.conf.size(0) #removed len()
    )

### Willingness to contribute

The FiftyOne Community encourages bug fix contributions. Would you or another
member of your organization be willing to contribute a fix for this bug to the
FiftyOne codebase?

-   [x ] Yes. I can contribute a fix for this bug independently
-   [ ] Yes. I would be willing to contribute a fix for this bug with guidance
        from the FiftyOne community
-   [ ] No. I cannot contribute a bug fix at this time

**Comment by swheaton** [comment]
This PR was merged recently that does exactly what you propose. Thank you for the suggestion! It will be out in 0.25.1
https://github.com/voxel51/fiftyone/pull/4720

**Comment by lyf6** [comment]
@swheaton have you solved this problem?

**Comment by lyf6** [comment]
@stevensagaert have you solved this problem?

**Comment by swheaton** [comment]
@lyf6 see my comment above, it will be fixed in 0.25.1

**Comment by lyf6** [comment]
@swheaton great! thank you for your reply
